modules/serverless-invoking-go-functions-cloudevent.adoc

```diff
@@ -6,7 +6,7 @@
 [id="serverless-invoking-go-functions-cloudevent_{context}"]
 = Functions triggered by a cloud event
 
-When an incoming cloud event is received, the event is invoked by the link:https://cloudevents.github.io/sdk-go/[CloudEvents Golang SDK] and the `Event` type as a parameter.
+When an incoming cloud event is received, the event is invoked by the link:https://cloudevents.github.io/sdk-go/[CloudEvents Golang SDK]. The invocation uses the `Event` type as a parameter.
```

Comment:
Assuming you agree with "Go" vs. "Golang". If not, I still think it should be consistent. 😺 

Note the SDK refers to itself as a "Go SDK". https://github.com/cloudevents/sdk-go#go-sdk-for-cloudevents

```suggestion
When an incoming cloud event is received, the event is invoked by the link:https://cloudevents.github.io/sdk-go/[CloudEvents Go SDK]. The invocation uses the `Event` type as a parameter.
```